lib/ui/playground/playground_button_present_alert.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:farmsmart_flutter/ui/common/Alert.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+class PlaygroundButtonPresentAlertViewModel {
+  Function(Alert, BuildContext) listener;
+  Alert alert;
+
+  PlaygroundButtonPresentAlertViewModel({
+    this.listener,
+    this.alert,
+  });
+}
+
+class PlaygroundButtonPresentAlert extends StatelessWidget {
```

Comment:
I don't get why this exists..
Its a super specific to show and alert in a playground.....

Why not have a generic "TestModal" Widget that takes a title and a function, then we can use that for all model tests.

Reply:
Perfect, i can use the modal button that's on the Landing Page PR, so when that branch is merged, i can adapt it to make it general, and use it for the Switching profiles page too

Reply:
👍

Reply:
Solved!